src/scripts.js

```diff
@@ -1,11 +1,140 @@
+/* eslint-disable max-len */
+/* eslint-disable space-before-blocks */
 // This is the JavaScript entry file - your code begins here
 // Do not delete or rename this file ********
 
 // An example of how you tell webpack to use a CSS (SCSS) file
 import './css/styles.css';
+import Customer from './customer'
+import Booking from './booking'
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
+// import './images/turing-logo.png'
 
+// Endpoints
 
-console.log('This is the JavaScript entry file - your code begins here.');
+let bookingsURL = "http://localhost:3001/api/v1/bookings";
+let customersURL = "http://localhost:3001/api/v1/customers"
+let roomsURL = "http://localhost:3001/api/v1/rooms"
+
+// Global Data Variables
+
+let currentCustomer;
+
+// Query Selectors
+
+const customerWelcomeName = document.querySelector(".customer-name")
+const customerTotalSpentDisplay = document.querySelector(".total-spent .dollars")
+
+const individualBookingTemplate = document.querySelector(".individual-booking.template")
+const individualBookingsContainer = document.querySelector(".individual-bookings-container")
+
+// Event listeners
+
+window.addEventListener("load", start)
+
+function chooseRandomCustomerID() {
+  let randomIDNumber = Math.floor((1 + Math.random() * 50))
+  return randomIDNumber
+}
+
+function start() {
+  console.log("Here we go!") 
+  loadData()
+  console.log("Data loading")
+}
+
+function loadData(){
+  Promise.all( [fetch(customersURL), fetch(bookingsURL), fetch(roomsURL), fetch(customersURL + "/" + chooseRandomCustomerID())] )
```

Comment:
This could live in an apiCalls folder if you wanted, interpolating the urls could keep the fetch dynamic something like: 
`function fetchData(dataset) {
  return fetch('http://localhost:3001/api/v1/${dataset}')`